proposals/dark_mode_extension_icons.md

```diff
@@ -0,0 +1,199 @@
+# Dark Mode Extension Icons
+
+**Summary**
+
+Feature to enable developers to enhance extension icon visibility in dark mode.
+
+**Document Metadata**
+
+**Author:** solomonkinard
+
+**Sponsoring Browser:** Chromium, Safari, Firefox
+
+**Contributors:** oliverdunk, xeenon, carlosjeurissen
+
+**Created:** 2024-04-05
+
+**Related Issues:**
+* https://crbug.com/893175
+* https://github.com/w3c/webextensions/issues/229
+
+## Motivation
+
+### Objective
+
+Extension developers will be able to supply and define a set of icons to be used
+in the event that user has expressed the preference for a page that has a dark
+theme. The set of dark mode icons can and will be made automatically visible
```

Comment:
Thoughts on dropping the second line here? We could also keep it, but if we do that I think we should remove the mention of a "system setting" to avoid dictating what the browser uses when determining the preferred mode.